### PR TITLE
fix: Resolve Settings fragment malfunctioning

### LINKED ISSUE #2055: Settings page malfunctions on moving back from about us
**Describe the bug**
Settings page shows malfunctioning on moving back from about us to settings .Many features including login/logout do not work properly.

**Expected Behaviour**

Settings page should work  as normal as it was working previously.

**Steps to reproduce it**
change in chat setttings fragment

**LogCat for the issue**

**Screenshots of the issue**
Screenrecording_20190322_030236 (1)

**Smartphone Info:**

|               | |
|---------------|-|
|Device         |Vivo 1802|
|Android Version|8.1.0    |

**Would you like to work on the issue?**
yes

### PR DESCRIPTION
**Pull request by chanmol1999**
Fixes #2055 

Changes: Chat settings fragment class

Screenshots for the change: 
Screenrecording_20190322_221614

### CODE REVIEW

File: app/src/main/java/org/fossasia/susi/ai/skills/settings/ChatSettingsFragment.kt [hunk 1]
```diff
@@ -249,6 +249,8 @@ class ChatSettingsFragment : PreferenceFragmentCompat(), ISettingsView {
 
     override fun onResume() {
         val thisActivity = activity
+        settingsPresenter = SettingsPresenter(activity as SkillsActivity)
+        settingsPresenter.onAttach(this)
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/susi/ai/skills/settings/ChatSettingsFragment.kt]: This is a workaround. Not a fix

Comment (chanmol1999) [reply app/src/main/java/org/fossasia/susi/ai/skills/settings/ChatSettingsFragment.kt]: Then what can I do for a fix . Please help sir.

**Comment by iamareebjamal** [comment]
Don't show login if already logged in

File: app/src/main/java/org/fossasia/susi/ai/skills/settings/ChatSettingsFragment.kt (hunk 1 above)
Comment (chanmol1999) [reply app/src/main/java/org/fossasia/susi/ai/skills/settings/ChatSettingsFragment.kt]: @iamareebjamal sir by this i will be able to fix login functions if any other malfunctions then?

Comment (iamareebjamal) [reply app/src/main/java/org/fossasia/susi/ai/skills/settings/ChatSettingsFragment.kt]: Don't show login if already logged in

**Comment by arundhati24** [comment]
@chanmol1999 Status of this PR?

**Comment by chanmol1999** [comment]
> @chanmol1999 Status of this PR?

@arundhati24  I am working on it.

**Pull request closed without merging**